aws/protocol/query/array.go

```diff
@@ -0,0 +1,37 @@
+package query
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// Array represents the encoding of Query lists and sets
+type Array struct {
+	values url.Values
+	prefix string
+	flat bool
+	memberName string
+	size int32
+}
```

Comment:
I don't know what prefix member refers to for query. Is this like the xml headers prefix?